ios/FluentUI/Bottom Sheet/BottomSheetTokenSet.swift

```diff
@@ -16,7 +16,8 @@ public class BottomSheetTokenSet: ControlTokenSet<BottomSheetTokenSet.Tokens> {
         super.init { token, theme in
             switch token {
             case .backgroundColor:
-                return .dynamicColor { theme.aliasTokens.colors[.background2]
```

Comment:
Is this so there's no difference in darkElevated?

Reply:
well.. bottomsheet doesnt have a darkmode. it should be really only darkelevated because the nature of bottomsheet. similar thing for drawer or popover background color